pkgs/applications/networking/owncloud-client/default.nix

```diff
@@ -1,16 +1,32 @@
-{ lib, fetchurl, mkDerivation, cmake, extra-cmake-modules, pkg-config, qtbase, qtkeychain, sqlite, libsecret }:
+{ lib
+, fetchFromGitHub
+, mkDerivation
+, pkg-config
+, cmake
+, extra-cmake-modules
+, callPackage
+, qtbase
+, qtkeychain
+, qttools
+, sqlite
+, libsecret
+}:
 
 mkDerivation rec {
   pname = "owncloud-client";
-  version = "2.11.0.8354";
+  version = "3.2.1";
 
-  src = fetchurl {
-    url = "https://download.owncloud.com/desktop/ownCloud/stable/${version}/source/ownCloud-${version}.tar.xz";
-    sha256 = "sha256-YraWvGgeF5b1+3i5Jlk+bwvAULr7KFOSX8y0ZoPpljI=";
-  };
+  libregraph = callPackage ./libre-graph-api-cpp-qt-client.nix { };
+
+  src = (fetchFromGitHub {
+    owner = "owncloud";
+    repo = "client";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-39tpvzlTy3KRxg8DzCQW2VnsaLqJ+dNQRur2TqRZytE=";
+  });
```

Comment:
Parenthesis here (and in other file) aren't necessary, but that's just a minor issue.
Otherwise I think this looks good for a merge.

Reply:
aha right, noted, will refactor when 4.0.0 is out.